test/include/RAJA_unit_forone.hpp

```diff
@@ -5,20 +5,114 @@
 // SPDX-License-Identifier: (BSD-3-Clause)
 //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
 
+#ifndef RAJA_unit_forone_HPP
+#define RAJA_unit_forone_HPP
+
+#include <RAJA/RAJA.hpp>
+
+#include <type_traits>
+
 ///
-/// Header file of macro for-one CUDA unit tests.
-/// Use as:
-/// forone<<<1,1>>>( [=] __device__ () {} );
+/// forone_pol<forone_policy>( [=] RAJA_HOST_DEVICE(){ /* code to test */ } );
```

Comment:
If the old forone<<<>>> has been removed, can you rename forone_pol -> forone?

Reply:
Done. Wasn't sure if you wanted that before because it might affect the sort PR.

Reply:
Its probably better to fix it now.